coptic_RNN.py

```diff
@@ -105,7 +106,7 @@ def mask_and_label_characters(self, data_item):
                     replacement = self.mask
                 elif r2 < 0.9:
                     # replace with random character
-                    replacement = random.randint(100, self.num_tokens - 1)
+                    replacement = random.randint(3, self.num_tokens - 1)
```

Comment:
should start with 0 so the 4 special ones we have are 0-3 (checked with a print in the sentence piece file in case we make any updates)